### ISSUE TITLE
With multiple changes, the documentation is not clear what happens from a transactional point of view.

### ISSUE THREAD

**Issue by adamsd5** [dotnet/EntityFramework.Docs#3773]
With multiple changes, the documentation is not clear what happens from a transactional point of view.  If I make two changes, and the second one fails in the database (for whatever reason), will the former be rolled back?  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3a489ad3-46e5-865f-09f8-b9be45788b8b
* Version Independent ID: e38f9319-2483-f96f-1d12-51c1489514e5
* Content: [DbContext.SaveChanges Method (System.Data.Entity)](https://docs.microsoft.com/en-us/dotnet/api/system.data.entity.dbcontext.savechanges?view=entity-framework-6.2.0)
* Content Source: [EF/xml/System.Data.Entity/DbContext.xml](https://github.com/dotnet/EntityFramework.ApiDocs/blob/live/EF/xml/System.Data.Entity/DbContext.xml)
* Product: **entity-framework**
* Technology: **entity-framework-6**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **avickers**

**Comment by roji** [comment]
Yes - SaveChanges is generally transactional, for database that support it: relational databases wrap the entire SaveChanges in a transaction (or use a savepoint), but some databases exist which don't support transactions.

We don't include full, exhaustive data in the API documentation - this is covered in the [conceptual docs](https://docs.microsoft.com/en-us/ef/core/saving/basic#multiple-operations-in-a-single-savechanges).
